CMakeLists.txt

```diff
@@ -44,6 +45,9 @@ set_property(TARGET ${BIN_TARGET} PROPERTY CXX_STANDARD_REQUIRED ON)
 set_property(TARGET ${BIN_TARGET} PROPERTY CXX_STANDARD 11)
 set_property(TARGET ${BIN_TARGET} PROPERTY CXX_EXTENSIONS OFF)
 
+message(STATUS "webp include dir = ${WEBP_INCLUDE_DIR}")
+message(STATUS "webp lib = ${WEBP_LIBRARY}")
```

Comment:
Are these debug prints really needed?

Reply:
No, I forgot to take those out, sorry. Will do another commit for this.